mettle/configure.ac

```diff
@@ -12,7 +12,7 @@ AM_PROG_CC_C_O
 AC_PROG_LIBTOOL
 LT_INIT
 
-CFLAGS="$CFLAGS -Wall -std=gnu99"
+CFLAGS="$CFLAGS -Wall -Werror -std=gnu99"
```

Comment:
A warning I got while developing was 'calling a function with the wrong parameters'. Which I overlooked then had some strange failures until I did 'make clean'. This is intended to avoid the problem.